Explorer/Assets/Scripts/ECS/StreamableLoading/GLTF/GltFastDownloadProvider.cs

```diff
@@ -18,29 +18,34 @@ namespace ECS.StreamableLoading.GLTF
 {
     internal class GltFastDownloadProvider : IDownloadProvider, IDisposable
     {
-        public string TargetGltfOriginalPath = string.Empty;
-
+        private string targetGltfOriginalPath = string.Empty;
         private ISceneData sceneData;
         private World world;
         private IPartitionComponent partitionComponent;
         private const int ATTEMPTS_COUNT = 6;
 
-        public GltFastDownloadProvider(World world, ISceneData sceneData, IPartitionComponent partitionComponent)
+        public GltFastDownloadProvider(World world, ISceneData sceneData, IPartitionComponent partitionComponent, string targetGltfOriginalPath)
         {
             this.world = world;
             this.sceneData = sceneData;
             this.partitionComponent = partitionComponent;
+            this.targetGltfOriginalPath = targetGltfOriginalPath;
         }
 
         public async Task<IDownload> RequestAsync(Uri uri)
         {
+            // Even if the file is not found in the scene mappings, it can still be a valid uri, e.g. the already hashed GLTF file.
```

Comment:
I tried clarifying [this comment from unity-renderer](https://github.com/decentraland/unity-renderer/blob/5092dfe44d59e4b455a7473a5efc611eab6e04a6/unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/AssetManager/GLTFast/Wrappers/GLTFastDownloadProvider.cs#L68) since it's the same thing that we did in there.

This is what's happening:
* both `RequestTextureAsync()` and `RequestAsync()` need to get the final real url from the scene mappings with `sceneData.SceneContent.TryGetContentUrl()` because both are used for downloading GLTF external dependencies (like textures).
* `RequestAsync()` apparently is also used for fetching the GLTF/GLB file itself.... and for some reason that url **already comes with the asset hash, so it was already parsed from the mappings in some external previous step**. So looking for the hash of the asset in the mappings doesn't make sense.

I know it's a bit confusing, I tried to clarify with that comment... since GLTFast works that way (uses `RequestAsync()` for the GLTF itself besides for other external dependencies) we have to use the unparsed url in that case.

Reply:
Also, regarding your concern on deployment, remember all this raw GLTF stuff is only for local scene development.
The fallback for production will be tackled in an uncertain distant future.

Reply:
I made a mini refactor to always receive the raw path in the `RequestAsync()` so that we can get rid of that comment, tackled here: https://github.com/decentraland/unity-explorer/pull/1670/commits/6ff5ca110823a78049017e899b8b5f1cbb2899f5